server/middlewares/validations/pagination.js

```diff
@@ -0,0 +1,40 @@
+import Joi from 'joi';
+
+/**
+ * @author Mutombo jean-vincent
+ * @class Pagination
+ * @description Handle validation for article pagination
+*/
+class Pagination {
+  /**
+   * @description Validate rating credentials in the request body
+   * @static
+   * @param {object} req
+   * @param {object} res
+   * @param {callback} next
+   * @returns {void}
+   */
+  static validatePagination(req, res, next) {
+    const schema = {
+      limit: Joi.number()
+        .integer()
+        .min(1)
+        .max(20),
+      offset: Joi.number()
+        .integer()
+        .min(0),
+    };
+
+    const pagination = Joi.validate(req.query, schema);
```

Comment:
There is a validation middleware made by hadad in `middleware/validations/index.js` which takes only the Joi schema and handle the rest. It helps us to not repeat the same code when validating, we just create the schema. 

`const pagination = Joi.validate(req.query, schema);
if (pagination.error) {
      return res.status(400).json({
        message: pagination.error.details[0].message.replace(/[^a-zA-Z0-9 ]/g, ''),
      });
    }
    next();`

Would you mind taking a look at it

Reply:
> There is a validation middleware made by hadad in `middleware/validations/index.js` which takes only the Joi schema and handle the rest. It helps us to not repeat the same code when validating, we just create the schema.
> 
> `const pagination = Joi.validate(req.query, schema); if (pagination.error) { return res.status(400).json({ message: pagination.error.details[0].message.replace(/[^a-zA-Z0-9 ]/g, ''), }); } next();`
> 
> Would you mind taking a look at it

Yes, @dusmel showed me this middleware. I would like to use it instead but he is still customising it. I am waiting for it to be ready